lib/ConstantFraction.js

```diff
@@ -176,13 +204,34 @@ ConstantFraction.addConstantAndFraction = function(node) {
     newNode.args[0] = newFractionNode;
     newNode.args[1] = newConstNode;
   }
-  return NodeStatus.nodeChanged(changeType, oldNode, newNode);
+
```

Comment:
this function might be too long now? not sure if it makes sense to break it up though